db/migrate/20240409215614_create_activities.rb

```diff
@@ -0,0 +1,15 @@
+class CreateActivities < ActiveRecord::Migration[7.1]
+  def change
+    create_table :activities do |t|
+      t.string :address
+      t.string :description
+      t.float :lat
+      t.float :lon
+      t.string :type
```

Comment:
I am making a seed file to test the API in Postman, it looks like ActiveRecord does not like us naming a variable `type`, so we will need to change this to maybe `activity_type` instead